Symfony/CS/AbstractPhpdocTypesFixer.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS;
+
+use Symfony\CS\DocBlock\DocBlock;
+use Symfony\CS\DocBlock\Line;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Graham Campbell <graham@mineuk.com>
+ */
+abstract class AbstractPhpdocTypesFixer extends AbstractFixer
```

Comment:
not quite sure what you mean? I named it like this because it fixes types - the two fixers that extend it fix built in types and scalar types